cli/command/trust/signer_remove.go

```diff
@@ -0,0 +1,124 @@
+package trust
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/cli/cli/trust"
+	"github.com/docker/notary/client"
+	"github.com/docker/notary/tuf/data"
+	"github.com/spf13/cobra"
+)
+
+type signerRemoveOptions struct {
+	forceYes bool
+	images   []string
```

Comment:
we shouldn't need `images` or `signer` in this options block, I would prefer to pass`args[0]` and `args[1:]` directly to `removeSigner`